packages/jest-cli/src/TestNamePatternPrompt.js

```diff
@@ -102,12 +90,18 @@ module.exports = (
           const more = total - max;
           pipe.write(
             // eslint-disable-next-line max-len
-            `\n  ${chalk.dim(`\u203A and ${more} more ${pluralizeTest(more)}`)}`,
+            `\n  ${chalk.dim(
+              `\u203A and ${more} more ${pluralizeTest(more)}`,
+            )}`,
           );
         }
       } else {
         // eslint-disable-next-line max-len
```

Comment:
Oh yeah! Let's do that in follow-ups when we actually get rid of eslint rules etc.